docs/content/preview/architecture/docdb-replication/async-replication.md

```diff
@@ -303,21 +307,22 @@ Improper use can compromise replication consistency and lead to data divergence.
 
 #### Transactional Automatic mode
 
-- All connections to the source universe must be reset after setting up the replication. [#25853](https://github.com/yugabyte/yugabyte-db/issues/25853)
-- The GRANT statement is currently not replicated. [#26461](https://github.com/yugabyte/yugabyte-db/issues/26461)
-- Adding unique constraints is currently not supported. [#26167](https://github.com/yugabyte/yugabyte-db/issues/26167)
-- Table rewrites (ALTER TABLE commands that require structural changes to the table) on partitioned tables are currently not supported. [#26453](https://github.com/yugabyte/yugabyte-db/issues/26453)
 - Global objects like Users, Roles, and Tablespaces are not replicated. These objects must be manually created on the standby universe.
 - DDLs related to Materialized Views (CREATE, DROP, and REFRESH) are not replicated. You can manually run these on both universes by setting the YSQL configuration parameter `yb_xcluster_ddl_replication.enable_manual_ddl_replication` to `true`.
 - CREATE TABLE AS and SELECT INTO DDL statements are not supported. You can work around this by breaking the DDL into a CREATE TABLE followed by INSERT SELECT.
 - ALTER COLUMN TYPE, ADD COLUMN ... SERIAL, TRUNCATE, and ALTER LARGE OBJECT DDLs are not supported.
 - DDLs related to PUBLICATION and SUBSCRIPTION are not supported.
-- While Automatic mode is active, you can only CREATE, DROP, or ALTER the following extensions: file_fdw, fuzzystrmatch, pgcrypto, postgres_fdw, sslinfo, uuid-ossp, hypopg, pg_stat_monitor, and pgaudit. All other extensions must be created _before_ setting up automatic mode.
+- Replication of colocated tables is not yet supported.
+- Rewinding of sequences (e.g., restarting a sequence so it will repeat values) is discouraged because it may not be fully rolled back during unplanned failovers.
```

Comment:
```suggestion
- Rewinding of sequences (for example, restarting a sequence so it will repeat values) is discouraged because it may not be fully rolled back during unplanned failovers.
```